.docker/Dockerfile

```diff
@@ -1,68 +1,82 @@
-ARG SCANCONTROL_SDK_VERSION=0.2.5
+ARG SCANCONTROL_SDK_VERSION=1.0.0
 
-FROM ubuntu:bionic AS build
+FROM ubuntu:focal AS build
 
 ARG SCANCONTROL_SDK_VERSION
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
       checkinstall \
+      cmake \
       debhelper \
       g++ \
       gcc \
+      gettext \
       git \
+      gobject-introspection \
+      gtk-doc-tools \
       intltool \
+      libgirepository1.0-dev \
       libglib2.0-dev \
+      libgstreamer1.0-dev \
+      libgstreamer-plugins-base1.0-dev \
+      libgstreamer-plugins-good1.0-dev \
+      libgtk-3-dev \
+      libusb-1.0-0-dev \
       libxml2-dev \
-      meson \
+      python3-pip \
+      ninja-build \
       pkg-config \
       unzip \
       wget \
-    && rm -rf /var/lib/apt/lists/*
+      xsltproc \
+    && rm -rf /var/lib/apt/lists/* && \
```

Comment:
I'd suggest expanding this a bit further - this is the standard 'clean-up' stanza I use:

```
apt-get autoremove -y &&\
apt-get clean -y &&\
rm -rf /var/lib/apt/lists/*
```